src/Model.js

```diff
@@ -626,7 +626,9 @@ export class Model {
             throw new OneTableArgError('Remove cannot retry', {properties})
         }
         params.parse = true
-        let items = await this.find(properties, params)
+        let findParams = Object.assign({}, params);
+        delete findParams.transaction;
+        let items = await this.find(properties, findParams)
```

Comment:
I was getting `Unknown transaction operation find` error because `removeByFind` is using the same params.